test/util/http/common.js

```diff
@@ -4,9 +4,8 @@ import Modal from '../../../src/components/modal.vue';
 import Spinner from '../../../src/components/spinner.vue';
 
 import { relativeUrl } from '../request';
-import { withAuth } from '../../../src/util/request';
 
-const assertRequestsMatch = (actual, expected, component) => {
+const assertRequestsMatch = (actual, expected) => {
```

Comment:
I really like that we don't have to pass in the component anymore: it feels cleaner like this. `assertRequestsMatch()` is used in two places in this file. How about updating those calls to remove the component?